SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -0,0 +1,156 @@
+//==------- ctor_default.cpp  - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for esimd default constructor.
+//
+// The simd constructor can't be constructed with sycl::half data type. The
```

Comment:
Minor/non-blocking comment:
```suggestion
// The simd can't be constructed with sycl::half data type. The
```
or
```suggestion
// The simd constructor can't be called with sycl::half data type now. The
```

Reply:
I've applied this comment was applied in this commit: 11cca0a5e06e3913eafa264395c723e135c1636c